src/scripts/examples.block.ts

```diff
@@ -1,4 +1,4 @@
-import 'isomorphic-fetch';
+import fetch from 'node-fetch';
```

Comment:
Has trouble mocking `isomorphic-fetch`, which really is just a polyfill wrapper for `node-fetch`. So decided to just use that directly.